### ISSUE TITLE
Cannot overwrite read-only variable

### ISSUE THREAD

**Issue by h0pes** [oh-my-fish/theme-bobthefish#342]
Just after an upgrade to `fish 3.6.1` on `arch 6.2.8-arch1-1 x86_64`, I,m getting the following error:

```bash
read: hostname: cannot overwrite read-only variable
~/.local/share/omf/themes/bobthefish/functions/fish_title.fish (line 11): 
            uname -n | read -l hostname __
            ^
in function '__bobthefish_title_user'
	called on line 18 of file ~/.local/share/omf/themes/bobthefish/functions/fish_title.fish
in function 'fish_title'
in command substitution
```

Theme is updated (using omf).

**Comment by h0pes** [comment]
Nevermind, issue gone after a reboot. Sorry

**Comment by bobthecow** [comment]
Yup, that was fixed in 76cac812064fa749ffc258a20398c6f6250860c5 🙂
